catalog/ui/catalog-ui-search/src/main/webapp/js/controllers/cesium.layerCollection.controller.js

```diff
@@ -96,10 +96,14 @@ define(['underscore',
         setAlpha: function (model) {
             var layer = this.layerForCid[model.id];
             layer.alpha = model.get('alpha');
+            layer.show = this.showLayer(model);
```

Comment:
Instead of adding the check here, we should update `common.layerCollection.controller.js` to call setShow when the alpha changes.

We could update this line: https://github.com/codice/ddf/blob/master/catalog/ui/catalog-ui-search/src/main/webapp/js/controllers/common.layerCollection.controller.js#L26
to be 
`this.listenTo(this.collection, 'change:show change:alpha', this.setShow);`